src/connection-manager/chains.ts

```diff
@@ -233,6 +240,12 @@ class Chains {
       if (skipped > 1 && skipped < 20) {
         chainWithThisValidator.incomplete = true
         addLedgerToChain(ledger, chainWithThisValidator)
+      } else {
+        log.info(
+          `DebugSkipped2:${`${next}:${ledger.ledger_index}`}:${Array.from(
```

Comment:
It might be easier to debug this data with `chainWithThisValidator.id`. The set of validators is probably identifying the public keys, which are not very informative (for human eyes).